added OAIE Sketch to ui editors

ZPiDER666 opened the pull request.

philsturgeon commented:
Ooo very interesting! We'll leave this here for a 3-4 weeks to let it mature a tad before we send folks to it, but I like the direction!

ZPiDER666 commented:
@philsturgeon , i've fixed some initial troubles and added some features plus an online demo - i think now it should be mature enough to get some traffic and hopefully find some contributors!
